### DISCUSSION TITLE
How does NativeWind compare to Tailwind React Native Classnames?

### DISCUSSION THREAD

**Discussion by dakshshah96** [nativewind/nativewind#253] [General]
I'm currently setting up a new project in which I'd like to use Tailwind CSS with React Native and come across NativeWind's latest releases - great work!

I was wondering how NativeWind compares to [Tailwind React Native Classnames](https://github.com/jaredh159/tailwind-react-native-classnames/). Came across a [previous discussion](https://github.com/jaredh159/tailwind-react-native-classnames/discussions/171) on this, but does anything change with the recent NativeWind releases?

Thanks!

**Comment by marklawlor** [comment]
Currently that comparison is still accurate, but keep watching this space as I'll currently working on some big new features :100:

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
